packages/nifti-volume-loader/src/createNiftiImageIdsAndCacheMetadata.ts

```diff
@@ -25,7 +25,7 @@ export async function fetchArrayBuffer({
   onHeader,
   loadFullVolume = false,
 }) {
-  const isCompressed = url.endsWith('.gz');
+  const isCompressed = url.includes('.gz');
```

Comment:
but then how are you cleaning the url for the fetch?

Reply:
I don't need to clean the url. In my case, I use a signed URL and additional parameters are needed for authentication.

Reply:
Maybe use `URL` api instead

```js
const url = "https://toto.blob.core.windows.net/test5/myfoo1.gz?sv=2019-10-10&ss=bfqt&srt=sco&sp=rwdlacupx&se=2020-07-17T10:09:48Z&st=2020-07-17T02:09:48Z&spr=https&sig=xxxxxxxxx";
const _url = new URL(url);
const isCompressed = _url.pathname.endsWith('.gz');
console.log(isCompressed);


const url = "https://myserver.com/sample.gz";
const _url = new URL(url);
const isCompressed = _url.pathname.endsWith('.gz');
console.log(isCompressed);
```

Reply:
@daker good suggestion thanks! Updated the code.